mne/preprocessing/tests/test_maxwell.py

```diff
@@ -193,9 +192,15 @@ def test_multipolar_bases():
         assert_allclose(S_tot_mat, S_tot_mat_round, atol=1e-7)
         assert_allclose(S_tot_complex, S_tot_mat, rtol=1e-4, atol=1e-8)
 
+        # Now test our optimized version
+        S_tot = _sss_basis_basic(origin, coils, int_order, ext_order)
+        S_tot_fast = _sss_basis(origin, coils, int_order, ext_order)
+        # there are some sign differences in here...
```

Comment:
(the sign flip is uniform for a given column [or row, can't remember the orientation now])